asv_bench/benchmarks/utils.py

```diff
@@ -158,30 +154,16 @@ def gen_int_data(nrows: int, ncols: int, rand_low: int, rand_high: int) -> dict:
     dict
         Number of keys - `ncols`, each of them store np.ndarray of `nrows` length.
     """
-    cache_key = ("int", nrows, ncols, rand_low, rand_high)
-    if cache_key in data_cache:
-        return data_cache[cache_key]
-
-    logging.info(
-        "Generating int data {} rows and {} columns [{}-{}]".format(
-            nrows, ncols, rand_low, rand_high
-        )
-    )
```

Comment:
this doesn't seem like it has something to do with benchmarking true/false... can we split this up?

Reply:
Ok, unrelated changes were reverted, corresponding issue was created https://github.com/modin-project/modin/issues/3257.